code/modules/keybindings/bindings_client.dm

```diff
@@ -107,8 +107,3 @@
 	mob.focus?.key_up(_key, src)
 	mob.update_mouse_pointer()
 
-
-// Called every game tick
-/client/keyLoop()
-	holder?.keyLoop(src)
```

Comment:
What happened to this? Did it do anything?

Reply:
no, just extra proc overhead